hardened-chromium-source.spec

```diff
@@ -0,0 +1,59 @@
+%global numjobs %{_smp_build_ncpus}
+
+Name:	 hardened-chromium-source
+Version: 130.0.6723.91
+Release: clean
+Summary: Cleaned chromium source.
+Url:     http://www.chromium.org/Home
+License: BSD-3-Clause
+
+BuildRequires: git
+BuildRequires: python3
+
+%description
+Vanilla, cleaned, source of chromium. Used by hardened-chromium.
```

Comment:
And then this would just say "Vanilla chromium source tarball with proprietary bits removed"